protocols/ampleforth/index.json

```diff
@@ -1,6 +1,11 @@
 {
-	"cname": "ampleforth",
-	"description": "",
-	"path": "ampleforth",
-	"isEnabled": true
+  "cname": "ampleforth",
+  "description": "AMPL is the first rebasing currency of DeFi. It is an uncollateralized financial primitive with an elastic supply that adjusts each day according to demand in the marketplace. Like Bitcoin, AMPL is non-dilutive but it has a long-term unit of account that unlocks safe denomination in contracts, like debt.",
+  "path": "ampleforth",
+  "type": "compoundish",
+  "isEnabled": true,
+  "discourseForum": {
+    "url": "https://https://forum.ampleforth.org",
+    "categoryId": "6",
+  },
```

Comment:
Hey @brandoniles, it looks like there's some trailing commas on lines 9-10 causing the validation script to fail. The `type` field has also been deprecated, so you can safely remove that from this file now.

Everything else looks good 👍 .